.github/workflows/gh-container-image.yml

```diff
@@ -0,0 +1,36 @@
+name: Update container image
+
+on:   
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: subspace-relayer
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build image
+        run: docker build . --file Dockerfile --tag $IMAGE_NAME
```

Comment:
Why no build image with `ghcr.io/subspace/subspace-relayer:latest` tag right away though? You wouldn't need to re-tag it in that case.

Reply:
If there are no other concerns, `ghcr.io/subspace/relayer:latest` would be more concise IMHO. That being said, we can have `subspace/node`, `subspace/farmer`, etc later.

Reply:
I agree, however, we are not putting `node` and `farmer` on ghcr as discussed yesterday

Reply:
Yeah, just said about the image naming, not saying whether they should be hosted on ghcr or dockerhub :P.

Reply:
`ghcr.io/subspace/relayer:latest` is fine too, I'm just trying to keep everything consistent:
* package name
* directory name
* container image name

If all those things are absolutely identical, it makes life so much easier sometimes.